Sources/LibAuk/Model/Seed.swift

```diff
@@ -32,6 +34,10 @@ public class Seed: Codable {
             a.append(.init(key: 3, value: CBOR.utf8String(name)))
         }
         
+        if !(passphrase?.isEmpty ?? true) {
+            a.append(.init(key: 4, value: CBOR.utf8String(passphrase!)))
+        }
```

Comment:
this may be easier to understand:
```
       if let passphrase = passphrase, !passphrase.isEmpty) {
            a.append(.init(key: 4, value: CBOR.utf8String(passphrase)))
        }
 ```